Feature/gui

yume-ind opened the pull request:
This PR creates the basic files for the Cassy gui, starting with the main VueClusters page. You can see some information about how to run the dev server by checking the `gui/README.md`

Commit:
created main page for cassy gui, modified entrypoint server for development

Commit:
added some info to the readme so people can run the dev environment

Torch3333 commented on entrypoint.go:
```diff
@@ -14,9 +14,11 @@ import (
 
 var (
 	endpoint = flag.String("cassy server endpoint", "localhost:20051", "endpoint of Cassy server")
+	mode     = flag.Bool("dev", false, "enable this option to run as dev server")
```
Should this var name be more explicit like `devMode` or something like this ?

yume-ind replied:
Ok, I can change the variable name.

Torch3333 reviewed (commented):
So far so good. 
Regarding the menu button, I am wondering if it won't be more practical to move it the left side.
At the moment, you click on the button which is on the right but it displays the contents on the left so I suppose this can be improve. What do you think ?

Commit:
added button to register new clusters

Commit:
changed position of navbar button

Commit:
fixed entrypoint variable

yume-ind commented:
@Torch3333 I changed the navbar position and fixed the variable. Also, I realized I had forgotten to add a Register Cluster button on the page, so I added that and changed the style of the other buttons.

Torch3333 commented:
Thanks for the changes. I think you can continue the development.
We will see with Yamada-san tomorrow morning if he wants to see the UI before everything is finished or not.
